core/unix/os_exports.h

```diff
@@ -246,8 +246,14 @@ ushort
 os_get_app_tls_reg_offset(reg_id_t seg);
 void *
 os_get_app_tls_base(dcontext_t *dcontext, reg_id_t seg);
+#if defined(AARCH64) || (defined(X64) && defined(X86))
+/* os_dump_core_live has the same restriction as dr_suspend_all_other_threads_ex().
+ * For X86_64 platform, fast FP save and restore (fxsave64) support is required. And mixed
+ * mode (a process mixing 64-bit and 32-bit code) is not supported.
```

Comment:
Good to have here too, but this needs to be in the user-face public doxygen docs for dr_create_memory_dump().  That's what generates the html docs at dynamorio.org.

Reply:
I've updated the doxygen docs in dr_tools.h.